myapp/spec/rails_helper.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+#require_relative '../config/environment'
+require File.expand_path('../config/environment', __dir__)
+abort('The Rails environment is running in production mode!') if Rails.env.production?
+require 'rspec/rails'
```

Comment:
requireはまとめた方が読みやすいかなと思いました。
あとコメントは不要であれば削除をお願いします。

```suggestion
require 'spec_helper'
require 'rspec/rails'
require File.expand_path('../config/environment', __dir__)

ENV['RAILS_ENV'] ||= 'test'
abort('The Rails environment is running in production mode!') if Rails.env.production?
```

Reply:
順番を変更したところ、エラーとなったのでそのままにしました。
コメントは削除しました。